src/lemlib/chassis/opcontrol.cpp

```diff
@@ -35,21 +35,26 @@ void Chassis::tank(int left, int right, bool disableDriveCurve) {
  * @param turn speed to turn. Takes an input from -127 to 127.
  * @param disableDriveCurve whether to disable the drive curve or not. If disabled, uses a linear curve with no
  * deadzone or minimum power
+ * @param desaturateBias how much to favor turning over forward/backward motion or vice versa when motors are
+ * saturated. A value of 0 fully prioritizes driving, a value of 1 fully prioritizes turning
  */
-void Chassis::arcade(int throttle, int turn, bool disableDriveCurve) {
+void Chassis::arcade(int throttle, int turn, bool disableDriveCurve, float desaturateBias) {
     // use drive curves if they have not been disabled
     if (!disableDriveCurve) {
         throttle = throttleCurve->curve(throttle);
         turn = throttleCurve->curve(turn);
```

Comment:
Because of the nature of the drive curves, they may return non-integer values. In this case we should round the outputs to the nearest integer

```suggestion
        throttle = std::round(throttleCurve->curve(throttle));
        turn = std::round(throttleCurve->curve(turn));
```